geospaas/nansat_ingestor/tests.py

```diff
@@ -112,10 +111,15 @@ def setUp(self):
         # in order to prevent "mock leak" in the tests
         self.addCleanup(self.patcher.stop)
```

Comment:
Teardown is not really needed. It can be replace with self.addCleanup(patch.stopall) in setUp.

```suggestion
        self.addCleanup(patcher.stopall)
```

Reply:
`self.addCleanup` can also be moved down after all patchers.